editor/editor_export.cpp

```diff
@@ -1028,6 +1028,10 @@ Error EditorExportPlatform::_add_shared_object(void *p_userdata, const SharedObj
 Error EditorExportPlatform::save_pack(const Ref<EditorExportPreset> &p_preset, const String &p_path, Vector<SharedObject> *p_so_files, bool p_embed, int64_t *r_embedded_start, int64_t *r_embedded_size) {
 	EditorProgress ep("savepack", TTR("Packing"), 102, true);
 
+	// Create the temporary export directory if it doesn't exist.
+	DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
```

Comment:
You should always `memdelete()` a `DirAccess *`, otherwise that's a memory leak. Or better, use `DirAccessRef` which is ref-counted.